docs/installationUbuntu20.04.md

```diff
@@ -0,0 +1,169 @@
+# Tested on Ubuntu 18.04.4 LTS (bionic Beaver) and 20.04 LTS (Focal Fossa)
+
+## Prerequisites
+* yarn
+```
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list 
+sudo apt update && sudo apt install yarn
+```
+* Rust
+```
+curl https://sh.rustup.rs -sSf | sh
+```
+* Cargo
+```
+sudo apt install cargo
+```
+
+## Update ubuntu
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get dist-upgrade
+```
+
+## Install dependencies
+```
+sudo apt-get install git ssh curl wget gcc g++ cmake \
+libstxxl-dev libxml2-dev libsparsehash-dev libbz2-dev zlib1g-dev libzip-dev libgomp1 \
+pkg-config libgdal-dev libtbb-dev psmisc build-essential postgresql postgis \
+clang libboost-all-dev libexpat1-dev libjsoncpp-dev libncurses5-dev lua5.3 liblua5.3-dev \
+powerline fonts-powerline zsh
+```
+
+## Create sources directory
+```
+cd ~
+mkdir sources
+```
+
+## Install and compile capnproto
+```
+cd sources
```

Comment:
Also, no libcapnp-dev in ubuntu??

Reply:
Indeed, maybe that was so before, but the trRouting dockerfile just installs the packages, so I guess it will work.